README.md

```diff
@@ -60,20 +54,19 @@ Most likely you didn't get that sick dev job, and are doing QA, or IT, or someth
 
 ### The offer
-If you do get an offer: Negotiate. They will _never_ take away your offer because you tried to negotiate. Always ask for a few dollars more, especially if you have a competing offer. Just ask nicely, "Hey, could you increase the salary from 20 to 24 an hour? I would feel a lot more confident about my decision if you could do that for me" or "I have an offer from AutoSoft, and while I would love to work with you, I couldn't justify it if I could get more there". If they say something like "HR won't let me" that's a lie, but you probably don't have enough leverage to negotiate anyway. I used to negotiate for jobs (and return offers) I knew I wasn't going to take just to practice or get better leverage. This advice is more important for fulltime, but it's good to get into the habit now.
+If you do get an offer: negotiate. They will _never_ take away your offer because you tried to negotiate. Always ask for a few dollars more, especially if you have a competing offer. Just ask nicely, "Hey, could you increase the salary from 20 to 24 an hour? I would feel a lot more confident about my decision if you could do that for me" or "I have an offer from AutoSoft, and while I would love to work with you, I couldn't justify it if I could get more there". If they say something like "HR won't let me" that's a lie, but you probably don't have enough leverage to negotiate anyway. I used to negotiate for jobs (and return offers) I knew I wasn't going to take just to practice or get better leverage. This advice is more important for fulltime, but it's good to get into the habit now.
 
 It can feel super scary to ask for more, especially if you feel lucky to even get the offer. I still get too scared, but in retrospect it's always worth it. More [info for fulltime here](https://www.kalzumeus.com/2012/01/23/salary-negotiation/).
 
 ### On the job
-Nice, you got a job! Dedicate yourself to this coop, ask as many questions as you can, and leech knowledge from your coworkers. Ask for projects you think are too hard for you, and then ask for help. Looking stupid doesn't matter if the first three months. There's this type of person that asks a lot of basic questions for months and before you know it, they know everything and you're coming to them for help. It's good to be this type of person.
+Good _job_, you got a job! Dedicate yourself to this coop, ask as many questions as you can, and leech knowledge from your coworkers. Ask for projects you think are too hard for you, and then ask for help. Looking stupid doesn't matter if the first three months. It's good to be the type of person that asks a lot of basic questions for months, and before you know it, they know everything and people are coming to them for help.
```

Comment:
no

Reply:
github does not have granularity within lines, please revert the "Good _job_" change. the "no" was supposed to be anti-pun humour but instead it looks like i'm brutally rejecting your edits.